### PR TITLE
REST API

### PR DESCRIPTION
**Pull request by lealal**
djangoapi
Created an API using the Django rest framework.

/api/tests -> view all tests 
/api/persons -> view all objects from the person table

https://www.django-rest-framework.org/tutorial/quickstart/

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 asgiref==3.2.10
 Django==3.1.2
-psycopg2-binary
+djangorestframework==3.12.2
+gunicorn==20.0.4
```
Comment (hocally) [inline requirements.txt:4]: @lealal Good idea to lock this down, do you know if this is the same version we are currently using?

File: data_entry/serializers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    class Meta:
+        model = Test
+        fields = ('weather','temperature','driver','location',
+                'track', 'fast_lap', 'tires', 'tire_condition',
+                'engine', 'software', 'comments','created_at','personnel')
+
+class PersonSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Person
+        fields = ('email', 'name', 'subteam', 'phone', 'licensed')
```
Comment (hocally) [inline data_entry/serializers.py]: Nit: files should end with a newline

File: data_entry/urls.py [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+router.register(r'persons', views.PersonViewSet)
 
 urlpatterns = [
     path('', views.home, name='home'),
     path('collectdata', views.collectData, name='collectdata'),
     path('view_data', views.viewData, name='view_data'),
     path('create_user', views.createUser, name='create_user'),
+    path('api/', include(router.urls)),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework'))
 ]
```
Comment (hocally) [inline data_entry/urls.py]: Nit: files should end with a newline

**Review by hocally — APPROVED** [review]
@lealal Looks clean, a few minor nits that would be nice to fix if you have to change something else as well. Posting a screenshot of what it looks like would be great

**Comment by hocally** [comment]
@lealal Would you mind fixing the README in this PR? I messed it up a while ago and might as well just throw it in here

**Comment by hocally** [comment]
Looks good to me, @nirbhayvig, any thoughts?

**Review by nirbhayvig — APPROVED** [review]

**Pull request merged**